boot/boot.go

```diff
@@ -169,6 +170,23 @@ type NameAndRevision struct {
 	Revision snap.Revision
 }
 
+// DebugDumpBootvars writes a dump of the bootvars to the given writer
+func DebugDumpBootvars(w io.Writer) error {
```

Comment:
don't think it needs a "Debug" prefix, OTOH I would put it a debug.go file by itself